### PR TITLE
Update FormSteps tests including a11y tests

### LINKED ISSUE #13: Render the QuestionForm
Implement functionality in React to retrieve catalog from Redux, parse the groups & questions.

Storing the state is being tracked in another issue and is not in scope here. This issue is just tracking the creation of a React-based representation of the catalog objects.

Handles displaying the relevant page, and navigation forward & backward

- QuestionForm contains a single Section component
  - passes in section_key, list of question_keys into Section component
  - Section reads & renders section info (title) from i18next using SectionKey
  - Section reads question type from Redux using questionKeys, in order to know which question component to render
- QuestionForm contains forward/back button, "Step # of #" label

### PR DESCRIPTION
**Pull request by mscharfstein**
Fixes #13 (by finishing the form tests)

Includes a11y testing but no tab navigation because there's really nothing to click in the FormSteps component - the parent Form component has the navigation buttons and the children Section component has the questions

### CODE REVIEW

File: client/src/components/FormSteps.test.js [hunk 1]
```diff
@@ -3,21 +3,110 @@ import { screen } from "@testing-library/react";
...
+test("form steps is accessible", async () => {
+  const form = [{ HOUSEHOLD_SECTION: ["HOUSEHOLD_SIZE", "ADULTS_65_PLUS"] }];
+  const { container } = renderWith(<FormSteps page={1} form={form} />);
+  expect(await axe(container)).toHaveNoViolations();
+});
+
+test("form steps is accessible with error", async () => {
+  const form = [{ HOUSEHOLD_SECTION: ["HOUSEHOLD_SIZE", "ADULTS_65_PLUS"] }];
+  const { container } = renderWith(
+    <FormSteps page={1} form={form} error={true} />
```
Comment (kebrahim) [inline client/src/components/FormSteps.test.js:109]: should this be showError instead of error?

File: client/src/components/FormSteps.test.js [hunk 2]
```diff
@@ -3,21 +3,110 @@ import { screen } from "@testing-library/react";
...
+        id: "integer-question-HOUSEHOLD_SIZE",
+      });
+      expect(question1).toBeInTheDocument();
+      const numErrorText = screen.getByText("error.question.number");
+      expect(numErrorText).toBeInTheDocument();
+    });
   });
 });
+
+test("form steps is accessible", async () => {
```
Comment (kebrahim) [inline client/src/components/FormSteps.test.js:100]: could these tests be moved inside the `describe` sections above? it seems like the FormSteps used are the same

File: client/src/components/FormSteps.test.js (hunk 1 above)
Comment (mscharfstein) [reply client/src/components/FormSteps.test.js:109]: yes thanks

File: client/src/components/FormSteps.test.js (hunk 2 above)
Comment (mscharfstein) [reply client/src/components/FormSteps.test.js:100]: i dont think so because we need access to the container object

**Pull request merged**